documentation/APIs/asyncapi3.yaml

```diff
@@ -0,0 +1,1211 @@
+      operationId: blocks
+      description: Publishes newly received blocks in their serialized binary form.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/transaction:
+    subscribe:
+      tags:
+        - name: blocks
+        - name: transactions
+      operationId: blocksTransaction
+      description: >-
+        Publishes newly received blocks containing a transaction payload in
+        their serialized binary form.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/transaction/tagged-data:
+    subscribe:
+      tags:
+        - name: blocks
+        - name: transactions
+      operationId: blocksTransactionsWithTaggedData
+      description: >-
+        Publishes newly received blocks containing a Tagged Data payload inside
+        transactions.
+      message:
+        $ref: '#/components/messages/Block'
+  'blocks/transaction/tagged-data/{tag}':
+    parameters:
+      tag:
+        description: >-
+          Hex encoded tag of the Tagged Data Payload inside the Transaction
+          Payload.
+        schema:
+          type: string
+          examples:
+            - '0xe45'
+    subscribe:
+      tags:
+        - name: blocks
+        - name: transactions
+      operationId: blocksTransactionsWithTag
+      description: >-
+        Publishes newly received blocks containing a Tagged Data payload inside
+        transactions with a specific tag.
+      message:
+        $ref: '#/components/messages/Block'
+  blocks/tagged-data:
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blocksWithTaggedData
+      description: >-
+        Publishes newly received blocks which contain tagged data payloads
+        (encoded in hex) in their serialized binary form.
+      message:
+        $ref: '#/components/messages/Block'
+  'blocks/tagged-data/{tag}':
+    parameters:
+      tag:
+        description: Hex encoded tag of the Tagged Data Payload.
+        schema:
+          type: string
+          examples:
+            - '0xe45'
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blocksWithSpecificTag
+      description: >-
+        Publishes newly received blocks which contain tagged data payloads with
+        the specified tag parameter (encoded in hex) in their serialized binary
+        form. Untagged data blocks do not get published.
+      message:
+        $ref: '#/components/messages/Block'
+  'block-metadata/{blockId}':
+    parameters:
+      blockId:
+        description: Hex encoded identifier of the block.
+        schema:
+          type: string
+          examples:
+            - '0xcf5f77d62285b9ed8d617729e9232ae346a328c1897f0939837198e93ec13e85'
+    subscribe:
+      tags:
+        - name: blocks
+      operationId: blockSpecificMetadata
+      description: Publishes metadata of a particular block whenever its metadata changes.
+      message:
+        $ref: '#/components/messages/BlockMetadata'
+  block-metadata/referenced:
```

Comment:
I don't think this event makes sense because what does `referenced` mean in the context of IOTA 2.0? Previously `referenced` meant that a block was referenced by a milestone and thus "included" but it is unclear what it means here.

Perhaps consider creating events for "confirmed" and "finalised"?

Reply:
`referenced` removed, and added `confirmed` and `finalized`.